bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/db/JdbcPostgresqlDAO.java

```diff
@@ -254,6 +297,19 @@ public void doStoreItemValue(Item item, State itemState, ItemVO vo, ZonedDateTim
         }
     }
 
+    @Override
+    public long doGetRowCount(String tableName) throws JdbcSQLException {
+        final String sql = StringUtilsExt.replaceArrayMerge(this.sqlGetRowCount, new String[] { "#tableName#" },
+                new String[] { tableName });
+        logger.debug("JDBC::doGetRowCount sql={}", sql);
+        try {
+            final @Nullable Long result = Yank.queryScalar(sql, Long.class, null);
+            return Objects.requireNonNullElse(result, 0L);
+        } catch (YankSQLException e) {
+            throw new JdbcSQLException(e);
+        }
+    }
+
```

Comment:
It looks like this method doesn't differ from the base implementation, so there should be no reason to override it?